xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.adm.sudo"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>sudo <!-- TODO: Better title --></title>
+  <subtitle><quote>With great power comes great responsibility</quote></subtitle>
+  <info>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker></dm:bugtracker>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+
+  <para>
+    Many commands and system utilities need to be run as &rootuser;
+    to modify files and/or perform tasks that only the super user is allowed to.
+
+    For security purposes and to avoid running dangerous commands accidentially,
+    it is generally disregarded to log in as &rootuser; directly.
+    Instead, it is recommended to work as a normal, unprivileged, user and
+    use the &sudo; command to run commands with elevated privileges.
+  </para>
+
+  <para>
+    In its default configuration, &sudo; is, apart of different options,
+    similiar to <command>su</command>.
+    Compared to <command>su</command>, &sudo; offers the possibility to allow users
+    to run commands with privileges of any other user in a highly configurable manner.
+    While <command>su</command> always uses PAM for authenticating as target user,
+    &sudo; makes it possible to authenticate with your own credentials.
+    This increases security by not having to share the &rootuser; password.
+
+    For instance, you can allow members of the group &examplegroup; to run a command <command>frobnicate</command>
+    as &exampleuserII;, with the restriction that no arguments are specified.
+    This can be used to assign roles with specific abilities to certain users and groups.
+  </para>
+
+  <sect1 xml:id="sec.adm.sudo.usage">
+    <title>Basic &sudo; usage</title>
+
+    <para>
+      &sudo; is simple to use, yet very powerful.
+    </para>
+
+    <sect2 xml:id="sec.adm.sudo.usage.cmd">
+      <title>Running a single command</title>
+      <para>
+        Logged in as normal user, you can run any command as &rootuser; by prepending it with &sudo;.
+        It will prompt for the root password and, if authenticated sucessfully, run the command as &rootuser;:
+      </para>
+
+      <screen>
+&prompt.user;id -un<co xml:id="co.sudo.usage.id"/>
+tux
+&prompt.user;sudo id -un
+root's password:<co xml:id="co.sudo.usage.pw"/>
+root
+&prompt.user;id -un
+tux<co xml:id="co.sudo.usage.after"/>
+&prompt.user;sudo id -un
+<co xml:id="co.sudo.usage.nopw"/>
+root
+</screen>
+
+      <calloutlist>
+        <callout arearefs="co.sudo.usage.id">
+          <para>The <command>id</command> command prints information about a user, the current one if not specified.</para>
```

Comment:
> id
uid=17237(fvogt) gid=50(suse) groups=50(suse),10(wheel),465(vboxusers)

That's more than just the login.